### ISSUE TITLE
bug(developer): the messages window should not be topmost when Keyman Developer is not focused

### ISSUE THREAD

**Issue by sewhite** [keymanapp/keyman#10562]
### Is your feature request related to a problem? Please describe.

When I am  in Keyman  Developer and then I switch to another application (for instance my  browser when I  want to test a touch keyboard), the "Messages" box that displays the results of my last compile stays does not minimize with the rest of Keyman Developer.
Image, after I switch to my browser after compiling a keyboard: 
![image](https://github.com/keymanapp/keyman/assets/16904461/9f8203ac-350f-40b6-b1ac-c03bed3cbf7a)

Furthermore, if I try to close the messages box, it minimizes the app I am in and takes me back to Keyman Developer.

### Keyman apps

- [ ] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [ ] Keyman for macOS
- [ ] Keyman for Windows
- [X] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

16.0.144

### Operating system

Windows 11 Home 22H2

### Device

Windows laptop

### Target application

browser,  WIndows settings, 

### Browser

Chrome, Edge

**Comment by mcdurdin** [comment]
You can drag and dock the messages window into the Developer main window -- does that do what you need?

**Comment by mcdurdin** [comment]
I am currently triaging this for future because we already have our feature set for 18.0 decided, but we will re-triage our Future issues in a few months, hopefully.

**Comment by sewhite** [comment]
Yes, dragging the messages window into the main window does solve the problem. It was not obvious that I could do that though. 
Also, I notice if I drag it into the main window, then drag it out again, it does not appear on top  of other applications for the rest of  that Keyman Developer session. This would be a useful default
